src/main/java/org/openstreetmap/atlas/geography/atlas/change/ChangeAtlas.java

```diff
@@ -60,7 +60,7 @@ public ChangeAtlas(final Atlas source, final Change change)
         }
         this.change = change;
         this.source = source;
-        new ChangeAtlasValidator(this).validate();
+        new AtlasValidator(this).validate();
```

Comment:
Not really. PackedAtlasBuilder makes checks at build time, and fails before the Atlas is created. Those checks expect an Atlas to be built already to be run. However, they are used in the ChangeAtlas right before exiting the constructor, having the same effective result.

Reply:
Ah right. But is there a reason we couldn't just validate the `PackedAtlasBuilder` at build time, all at once, using the a validator like this one? Instead of as things are added?